src/core/tests/type_prop/squeeze.cpp

```diff
@@ -639,4 +639,47 @@ TEST(SqueezeDynamicAxis, squeeze_dynamic_non_const_axes) {
     EXPECT_EQ(squeeze->get_allow_axis_skip(), allow_axis_skip);
 }
 
+TEST(SqueezeDynamicAxis, squeeze_dynamic_empty_axes) {
+    const auto p_shape = PartialShape{1, 2, -1, 4};
+    const auto axes_node = make_shared<ov::op::v0::Constant>(element::u64, Shape{});
+    const auto exp_shape = PartialShape{2, -1, 4};
+    const auto param = make_shared<ov::op::v0::Parameter>(element::f32, p_shape);
+
+    const auto squeeze0 = std::make_shared<op::v0::Squeeze>(param, axes_node);
+    const auto squeeze1 = std::make_shared<op::v15::Squeeze>(param, axes_node, true);
+    const auto squeeze2 = std::make_shared<op::v15::Squeeze>(param, axes_node, false);
```

Comment:
Make three tests each squeeze is different case (or parametrized tests) and give different test cases descriptive name

Reply:
Done